ui/tract-renderer/src/main/java/org/cru/godtools/tract/ui/controller/ParentController.kt

```diff
@@ -37,17 +37,19 @@ abstract class ParentController<T> protected constructor(
     // endregion Lifecycle
 
     // region Child Content
-    protected abstract val contentContainer: LinearLayout
+    protected abstract val contentContainer: ViewGroup
```

Comment:
What's the difference between a ViewGroup and the LinearLayout?

Reply:
ViewGroup is the base class for View that contains child views. Up until we added the `FallbackController` all of our renderer used `LinearLayout` to contain child elements. The `Fallback` rendering logic is using a `FrameLayout` instead, which necessitated widening the class for `contentContainer`